flapper-news/public/javascripts/posts/posts.factory.js

```diff
@@ -1,14 +1,14 @@
 // needed to inject $http
-app.factory('posts', ['$http', [function($http){
-	
+app.factory('posts', ['$http', function($http){
+
 	var o = {
 		posts: [],
 	};
-	
+
 	// to retrieve posts in the posts service within angularApp.js
-	o.getall = function(){
+	o.getAll = function(){
```

Comment:
You call `posts.getAll` in the `home` route's `resolve`.  This wasn't being called anywhere else, so I just changed it here.